File: python3/1066.py
qnt_par = 0
qnt_impar = 0
qnt_pos = 0
qnt_neg = 0

for index in range(1, 6):
    entrada = int(input())
    if(entrada % 2 == 0):
        qnt_par = qnt_par + 1
    else:
        qnt_impar = qnt_impar + 1
    if(entrada > 0):
        qnt_pos = qnt_pos + 1
    elif(entrada < 0):
        qnt_neg = qnt_neg + 1

print("{} valor(es) par(es)".format(qnt_par))
print("{} valor(es) impar(es)".format(qnt_impar))
print("{} valor(es) positivo(s)".format(qnt_pos))
print("{} valor(es) negativo(s)".format(qnt_neg))

File: c++/1066.cpp
#include <iostream>

using namespace std;

int main() {

    int n, i, par, impar, pos, neg;
    par = 0;
    impar = 0;
    pos = 0;
    neg = 0;

    for (i = 1; i <= 5; i++) {
        cin >> n;
        if (n % 2 == 0) {
            par = par + 1;
        } else {
            impar = impar + 1;
        }
        if (n > 0) {
            pos = pos + 1;
        } else if (n < 0) {
            neg = neg + 1;
        }
    }
    cout << par << " valor(es) par(es)" << endl;
    cout << impar << " valor(es) impar(es)" << endl;
    cout << pos << " valor(es) positivo(s)" << endl;
    cout << neg << " valor(es) negativo(s)" << endl;

    return 0;
}
